### ISSUE TITLE
Video is broken on Android in Kivy

### ISSUE THREAD

**Issue by germn** [kivy/kivy#4702]
- Currently [main](https://groups.google.com/forum/#!topic/kivy-users/ypJOZXlUEqM) and de facto only way to play video on Android in Kivy is to build project with `ffpyplayer`.
- `ffpyplayer` [needs](http://matham.github.io/ffpyplayer/installation.html#requirements) compiled for Android version of FFmpeg (and, btw, [doesn't](https://github.com/kivy/python-for-android/issues/464) have recipe for new p4a toolchain).
- Compiled for Android version of FFmpeg was provided by `ffmpeg` [recipe](https://github.com/kivy/python-for-android/tree/master/pythonforandroid/recipes/ffmpeg) based on [ffmpeg-android](https://github.com/WritingMinds/ffmpeg-android) project.
- Since `ffmpeg-android` project is [broken and dead](https://github.com/kivy/python-for-android/issues/920#issuecomment-257660141) we can't build `ffmpeg` and can't build `ffpyplayer` for Android.

Looks like for now it's impossible to play video on Android in Kivy at all.
Feel free to point if I'm wrong somewhere.

**Comment by germn** [comment]
Finally, I found [this thread](https://groups.google.com/forum/#!topic/kivy-users/ypJOZXlUEqM):

> TLDR: replace ffmpeg with ffpyplayer as a requirements for video support on Android and iOS.
> 
> Forget about your previous experience for Video on Android / iOS. The previous provider was made by myself during 2012, based on ffmpeg 1.2. It was largely buggy for some cases, and performance was not great.
> 
> This time is done. Our lovely core developer, matham, coded a new project named ffpyplayer (https://github.com/matham/ffpyplayer/), a cython implementation of a ffmpeg based player. And i just finished the Android recipes for it.
> 
> FFPyPlayer is now fully available on Android and iOS. Just add "ffpyplayer" as a recipe to compile, or in requirements for your buildozer. It will compile latest ffmpeg 2.7, and ffpyplayer.
> Also, since Kivy 1.9, it can be used as a core provider. Ie, just add it, and you'll get video support.

**Comment by KeyWeeUsr** [comment]
ref https://github.com/kivy/python-for-android/issues/920

**Comment by germn** [comment]
[Here's](https://github.com/kivy/python-for-android/issues/951) more on current situation.

Video on Android in Kivy supports by old toolchain only.

I think it might be good idea to add this information to Kivy docs.

**Comment by germn** [comment]
Possibly should be fixed by:
https://github.com/kivy/python-for-android/pull/952
https://github.com/kivy/python-for-android/pull/954
